addons/account/views/account_analytic_line_views.xml

```diff
@@ -18,7 +18,7 @@
                         <group/> <!-- put Accounting group under Amount group -->
                         <group name="accounting" string="Accounting">
                             <field name="general_account_id" options="{'no_create': True}"/>
-                            <field name="move_line_id" options="{'no_create': True}"/>
+                            <field name="move_line_id" options="{'no_create': True}" widget="open_move_line_move_widget"/>
```

Comment:
what about the one in the list view? Line 36

Reply:
Indeed, didn't think about it :+1: